src/System.Diagnostics.Process/tests/ProcessWaitingTests.cs

```diff
@@ -141,7 +143,7 @@ public void SingleProcess_CopiesShareExitInformation()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/corefx/issues/22269", TargetFrameworkMonikers.UapNotUapAot)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapNotUapAot, "Getting handle of child process running on UAP is not possible")]
```

Comment:
the issue is with remote invoker not specifically appcontainer - on UapAot it works slightly differently and we are actually able to do the right thing - @tarekgh has more context on that